plugins/modules/docker_image_load.py

```diff
@@ -0,0 +1,185 @@
+#!/usr/bin/python
+#
+# Copyright 2016 Red Hat | Ansible
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+DOCUMENTATION = '''
+---
+module: docker_image_load
+
+short_description: Load docker images from archives
+
+version_added: 1.3.0
+
+description:
+  - Load one or multiple Docker images from a C(.tar) archive, and return information on
+    the loaded images.
+
+options:
+  path:
+    description:
+      - The path to the C(.tar) archive to load Docker images from.
+    type: path
+    required: true
+
+extends_documentation_fragment:
+- community.docker.docker
+- community.docker.docker.docker_py_2_documentation
+
+notes:
+  - Does not support C(check_mode).
+
+requirements:
+  - "L(Docker SDK for Python,https://docker-py.readthedocs.io/en/stable/) >= 2.5.0"
+  - "Docker API >= 1.23"
+
+author:
+  - Felix Fontein (@felixfontein)
+
+'''
+
+EXAMPLES = '''
+- name: Load all images from the given tar file
+  community.docker.docker_image_load:
+    path: /path/to/images.tar
+  register: result
+
+- name: Print the loaded image names
+  debug:
+    msg: "Loaded the following images: {{ resut.image_names | join(', ') }}"
```

Comment:
```suggestion
    msg: "Loaded the following images: {{ result.image_names | join(', ') }}"
```

Reply:
Don't you know the famous `resut`? ;-) Thanks for spotting this!